cdap-ui/templates/ETLBatch/Elasticsearch.json

```diff
@@ -38,6 +38,21 @@
           "properties": {
             "width": "large"
           }
+        },
+        "schema" : {
+          "widget": "schema",
+          "label": "Schema",
+          "description" : "The schema or mapping of the data in elasticsearch.",
+          "schema-types" : [ "boolean", "int", "long", "float", "double", "bytes", "string" ],
```

Comment:
The user can use the CLI or a config file to create more complicated types. All schema types are supported.